python/img_hdf5_reader.py

```diff
@@ -107,6 +108,58 @@ def to_pngs(self, channel_clips, sample, json_file):
         # This JSON file is not used right now:
         # really just a list of the files processed.
 
+    def to_ometiff(self, channel, sample, ometif_path, clip, ometif_basename):
+        MAX_ALLOWED = 256
+        NP_TYPE = np.int8
+
+        scaled_sample = self.scale_sample(
+            channel=channel,
+            sample=sample,
+            max_allowed=MAX_ALLOWED,
+            clip=clip
+        ).astype(NP_TYPE)
+        return scaled_sample
+
+    def to_ometiffs(self, channel_clips, sample, json_file):
+        channels = {}
+        s3_target = open('s3_target.txt').read().strip()
+        images = []
+        for (channel, clip) in channel_clips:
+            channels[channel] = {
+                'sample': sample,
+                # TODO: Pass in portions of this path as parameters
+                'tileSource':
+                    'https://s3.amazonaws.com/'
+                + '{}/linnarsson.tiles/linnarsson.images.{}/'.format(
+                    s3_target, channel)
+                + 'info.json'
+            }
+            ometif_path = '{}.{}.ome.tif'.format(
+                json_file.name.replace('.json', ''),
+                channel
+            )
+            ometif_basename = basename(ometif_path)
+            array = self.to_ometiff(
+                channel=channel,
+                clip=float(clip),
+                sample=sample,
+                ometif_path=ometif_path,
+                ometif_basename=ometif_basename
+            )
+            images.append(array)
+
+        image = np.transpose(np.dstack(tuple(images)))
+        image = np.expand_dims(image, axis=0)
+        writer = omeTifWriter.OmeTifWriter(
+            file_path=ometif_path,
+            overwrite_file=True
```

Comment:
Is this necessary? Particularly when we're making bigger files, we really want to avoid accidentally clobbering them. Generally, in this project, we only run the code to generate a particular file if it isn't already present, so if just one file isn't coming out right, you can delete just that one, rerun, and only it is replaced.